### PR TITLE
Add composer normalize phive

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #155

### CODE REVIEW

**Comment by lukaszuznanski** [comment]
requires merge of #190

**Comment by oliverklee** [comment]
@lukaszuznanski I have merged #190. So you can rebase.

File: .gitlab/pipeline/.gitlab-ci.yml [hunk 1]
```diff
@@ -296,3 +297,11 @@ php-copypaste-check:
     - php-lint-php7.4
   script:
     - composer ci:php:copypaste
+
+composer-normalize:
+  extends: .default
+  stage: codestyle
+  needs:
+    - build-composer-dependencies
```
Comment (oliverklee) [inline .gitlab/pipeline/.gitlab-ci.yml:305]: This step does not need the Composer dependencies.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: .gitlab/pipeline/.gitlab-ci.yml (hunk 1 above)
Comment (lukaszuznanski) [reply .gitlab/pipeline/.gitlab-ci.yml:305]: that's true :)

**Review by oliverklee — APPROVED** [review]

**Review by oliverklee — APPROVED** [review]
Looks good. @lukaszuznanski Could you please rebase?

**Comment by lukaszuznanski** [comment]
@oliverklee ready

**Review by oliverklee — APPROVED** [review]

**Pull request merged**